how-to/samba/file-server.md

```diff
@@ -5,6 +5,12 @@ One of the most common ways to network Ubuntu and Windows computers is to config
 
 The server will be configured to share files with any client on the network without prompting for a password. If your environment requires stricter Access Controls see [Share Access Control](share-access-controls.md).
 
+```{warning}
+If you use **Samba and authd** at the same time, you must specify user and group mapping. Otherwise, you'd encounter permission issues due to mismatched user and group identifiers.
```

Comment:
```suggestion
If you use **Samba and authd** at the same time, you must specify user and group mapping. Otherwise, you will encounter permission issues due to mismatched user and group identifiers.
```